charts/rstudio-pm/values.yaml

```diff
@@ -110,12 +121,15 @@ pod:
   # -- the [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) for the pod
   securityContext: {}
   # -- the [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) for the main Package Manager container
+  #    Evaluated as a template.
```

Comment:
Nit: the fact that this is on another line creates a weird whitespace break in the generated `README`.